<p>Do not allow 'standing' before 'mountain' in code identifiers</p>

<h2>Noncompliant Code Example</h2>
<pre>
int standing = 1;
int mountain = 2; // Noncompliant
</pre>

<h2>Compliant Solution</h2>
<pre>
int mountain = 1;
int standing = 2;// Compliant
</pre>